File: public/codesamples/posts/graphes_grids/code3/Node.js
export default class Node {
  _x;
  _y;

  _connections = new Set();

  constructor(x, y) {
    this._x = x;
    this._y = y;
  }

  get x() {
    return this._x;
  }

  get y() {
    return this._y;
  }

  get connections() {
    return this._connections;
  }
}

File: public/codesamples/posts/graphes_grids/code3/Node.cs
namespace Posts.GraphesGrids.Code3
{
    public class Node
    {
        private int _x;
        public int x
        {
            get { return _x; }
        }
        private int _y;
        public int y
        {
            get { return _y; }
        }
        private HashSet<Connection> connections=new HashSet<Connection>();
        public HashSet<Connection> Connections
        {
            get { return connections; }
        }
        public Node(int x, int y)
        {
            _x = x;
            _y = y;
        }
    }
}
